packages/jobs/lib/execution/sync.ts

```diff
@@ -463,7 +463,7 @@ export async function abortSync(task: TaskSyncAbort): Promise<Result<void>> {
             logger.error(`failed to abort script for task ${task.abortedTask.id}: ${abortedScript.error}`);
         }
 
-        const syncJob = await isSyncJobRunning(task.syncId);
```

Comment:
we cannot get the running sync job since we update its status to STOPPED as soon as the user clicks on cancel. 
Don't remember why I did it this way originally, I guess I lazily used an existing function without thinking to much of the side effect